dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/WebContainer.java

```diff
@@ -211,6 +210,9 @@ public Object run() {
     
     // Servlet 4.0 : Must be static since referenced from static method
     protected static CacheServletWrapperFactory cacheServletWrapperFactory;
+    
+    //Servlet 6.0 - do not load it early
```

Comment:
What does this comment mean? Just that you're not initializing the variable here?